deploy/bicep/groups/monitoring.bicep

```diff
@@ -7,6 +7,9 @@ param environmentTag string
 param projectName string
 param location string
 
+param keyVaultName string
+param keyVaultResourceGroupName string
+
```

Comment:
These don't appear to be used, or I missed it if they are. Can these be removed if they're not needed?

Reply:
Removed the unused vars.